rubin_changelog/tag.py

```diff
@@ -44,13 +44,13 @@ def __init__(self, name: str):
         self._valid = False
         self._is_weekly = False
         self._is_main = False
-        self._hash = -1
         self._major = -1
         self._minor = -1
         self._patch = -1
         self._rc = -1
+        self._week = -1
+        self._yaer = -1
```

Comment:
Oops?

Reply:
Weird, 
It's correct in the merged version:
https://github.com/lsst-dm/lsst_git_changelog/blob/86ecad5b4092b943462682ee38705495b5a57c41/rubin_changelog/tag.py#L52
and my local version.